package.json

```diff
@@ -125,7 +125,7 @@
     "stylint": "^1.5.9",
     "stylus": "^0.54.0",
     "stylus-loader": "^2.4.0",
-    "svg-sprite-loader": "^0.0.31",
+    "svg-sprite-loader": "3.4.1",
```

Comment:
Do we still need this loader?

Reply:
No we don't, but I did not remove it in this PR because we need to do some work about tooling, like using create-cozy-app with cozy-scripts to manage dependencies, for instance.

Reply:
I update it anyway, because it was very outdated.

Reply:
Finally I reseted it... because it changed `yarn.lock` with a lot of new lines...